src/client/systems/sysstarsystemrenderer.cpp

```diff
@@ -277,7 +280,40 @@ void conquerspace::client::systems::SysStarSystemRenderer::SeeEntity() {
     });
 }
 
+void conquerspace::client::systems::SysStarSystemRenderer::DrawEntityName(
+    glm::vec3 &object_pos, entt::entity ent_id) {
+
+    if (m_app.GetUniverse().all_of<conquerspace::common::components::Name>(
+        ent_id)) {
```

Comment:
The formatting here is a bit wonky, do `using conquerspace::common::components::Name` and use name directly to reduce the number of namespace copying